psi4/driver/qcdb/basislistdunning.py

```diff
@@ -620,3 +620,108 @@ def load_basfam_dunning():
     basis_augccpvqz.add_dualfit('aug-cc-pvqz-dual')
     basis_heavyaugccpvtz.add_dualfit('heavy-aug-cc-pvtz-dual')
     basis_heavyaugccpvqz.add_dualfit('heavy-aug-cc-pvqz-dual')
+
+    basis_ccpvdz.add_guess('pcseg-0')
```

Comment:
What's the rationale behind using `pcseg-0` as a guess for the `cc` family (as well as for `zapa` below)? Are they both ecp-free?

Reply:
Yes, trying to match up ECP, designed for SCF vs. correlated, and puream characteristics better between guess orbital and target orbital basis sets. It was ECP that was the main problem (there's a trend there ...) when this was suggested. At the moment, the calc is stopping if guess/target ECP characteristics don't match up. This'll make that case less likely. IIRC, Susi suggested the pcseg as a modern ecp-free, correlation-designed family. I haven't used them yet myself.

Reply:
No, pcseg-0 was picked because it's a modern minimal polarization consistent basis that's similar in size to 3-21G but more accurate. It also covers the first four periods of the periodic table, where ECPs aren't necessary.